ReleaseProcessAutomation/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -38,7 +40,7 @@ public class ReleaseVersionAndMoveIssuesSubStep
     _jiraVersionReleaser = jiraVersionReleaser;
   }
 
-  public void Execute (SemanticVersion currentVersion, SemanticVersion nextVersion, bool squashUnreleased = false)
+  public bool Execute (SemanticVersion currentVersion, SemanticVersion nextVersion, bool squashUnreleased = false, bool movePreReleaseIssues = false)
```

Comment:
I want to understand why we need to return a `bool` here and explore other options. Am I correct in assuming this was the related part to the "user abort exception question"? :D 

I dislike the boolean return as it is unclear what the value indicates. If we want to let the use abort the whole step then we should do it using a custom exception that we catch at the bottom and ignore/log.

Reply:
Yeah, I didn't think it would get this much out of hand though.